typescript/packages/orb/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@subconsciousnetwork/orb",
+  "author": "Subconscious Inc.",
+  "license": "Apache-2.0 OR MIT",
+  "version": "0.1.0",
+  "description": "A batteries-included JavaScript package for accessing the Noosphere",
+  "type": "module",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "wireit",
+    "generate_wasm": "wireit",
+    "clean": "wireit",
+    "test": "wireit"
+  },
+  "dependencies": {
+    "@web/dev-server": "^0.1.34",
+    "esbuild": "^0.15.9",
+    "typescript": "~4.8.2",
+    "wireit": "^0.8.0"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "generate_wasm"
+      ],
+      "command": "tsc",
```

```suggestion
      "command": "tsc --pretty",
```

Since `tsc` won't think it's attached to a TTY when launched via wireit, it will turn off its nice colorful output by default.